src/dune_rules/install_rules.ml

```diff
@@ -110,11 +110,17 @@ end = struct
       (List.rev_concat_map
          ~f:(List.rev_map ~f:(fun f -> Section.Lib, f))
          (let { Mode.Dict.byte; native } = Lib_info.archives lib in
+          let jsoo_files =
+            (* A same runtime file can be used both for jsoo and wasmoo *)
```

Comment:
Does that comment still make sense now that jsoo and wasm options are different ?

Reply:
Yes, it still makes sense. In particular, one may want to pass a JavaScript file to wasm_of_ocaml to get some primitive purity information. Or one need to include some JavaScript code both when generating JavaScript and Wasm code. For instance, I have this for the `virtual_dom` package:
```
 (js_of_ocaml
  (javascript_files ../lib/virtualdom.compiled.js ./thunk.js ./hooks.js))
 (wasm_of_ocaml
  (javascript_files ../lib/virtualdom.compiled.js ./thunk.js ./hooks.js))
```